lib/firefly.ts

```diff
@@ -78,7 +78,11 @@ import HttpBase, { mapConfig } from './http';
 export default class FireFly extends HttpBase {
   private queue = Promise.resolve();
 
-  async getStatus(options?: FireFlyGetOptions): Promise<FireFlyStatusResponse> {
+  async getStatus(options?: FireFlyGetOptions): Promise<FireFlyStatusResponse | undefined> {
+    return this.getOne<FireFlyStatusResponse>(`/status`, options);
+  }
+
+  async getDefaultNamespaceStatus(options?: FireFlyGetOptions): Promise<FireFlyStatusResponse> {
     const response = await this.http.get<FireFlyStatusResponse>('/status', mapConfig(options));
```

Comment:
Won't this still query the namespace that was provided on client initialization instead of the default namespace? `this.http` is set to `${options.host}/api/v1/namespaces/${this.options.namespace}`. 

full code: https://github.com/hyperledger/firefly-sdk-nodejs/blob/eea0bbe70b5c92e1138078b9e123d95387d142a1/lib/http.ts#L40

Reply:
You are right. I misread the code.